viewing_party/party.py

```diff
@@ -1,23 +1,210 @@
-# ------------- WAVE 1 --------------------
+# # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+
+    if title == None or genre == None or rating == None:

```

Comment:
Here you are supposed to check if this is truthy but this only checks for None. To check if it's truthy you could so something like if not title or not genre or not rating